wp-parsely.php

```diff
@@ -105,6 +106,20 @@ function parsely_admin_menu_register(): void {
 	$settings_page->run();
 }
 
+require __DIR__ . '/src/class-recommended-content.php';
+
```

Comment:
To keep consistency with the rest of the file, could we remove the line break on 110?